### PR TITLE
port tests in test_functional_tensor to pytest

### PR DESCRIPTION
**Pull request by sahilg06**
Refactored the following tests to pytest in the file test/test_functional_tensor.py mentioned in issue #3956

Group A

- test_assert_image_tensor
- test_vflip
- test_hflip
- test_crop 

Group E

- test_gaussian_blur

### CODE REVIEW

File: test/test_functional_tensor.py
Comment (NicolasHug) [inline test/test_functional_tensor.py:1007]: It's no big deal but I find this formatting to be a bit cleaner, and easier to read:
```py
@pytest.mark.parametrize('func, args', [
    (F_t._get_image_size, ()),
    (F_t.vflip, ()),
    (F_t.hflip, ()),
    (F_t.crop, (1, 2, 4, 5)),
    (F_t.adjust_brightness, (0., )),
    (F_t.adjust_contrast, (1., )),
    ...
])
```

File: test/test_functional_tensor.py
Comment (NicolasHug) [inline test/test_functional_tensor.py:1062]: We could do something similar here

File: test/test_functional_tensor.py
Comment (NicolasHug) [inline test/test_functional_tensor.py:1116]: no need to skip lines here:

```suggestion
    if image_size == 'small':
        tensor = torch.from_numpy(
            np.arange(3 * 10 * 12, dtype="uint8").reshape((10, 12, 3))
        ).permute(2, 0, 1).to(device)
    else:
```

**Review by NicolasHug — APPROVED** [review]
Thanks @sahilg06 ! This looks great, I only have some minor cosmetics comments, but LGTM when green!

**Comment by sahilg06** [comment]
> Thanks @sahilg06 ! This looks great, I only have some minor cosmetics comments, but LGTM when green!

Thanks a lot @NicolasHug
 So should I do the suggested cosmetic changes or its fine to merge?

**Comment by NicolasHug** [comment]
> So should I do the suggested cosmetic changes

Yes ;), I tend to approve early when the requested changes are easy / obvious. I sometimes make comments that don't really need to be addressed and for those I usually say they're Nits. The  comments here were bordeline nits.

Thanks a lot for the PR!

**Pull request merged**
